config/config.go

```diff
@@ -27,14 +27,13 @@ import (
 // directly to YAML. Fields that cannot be set in the cloud-config (fields
 // used for internal use) have the YAML tag '-' so that they aren't marshalled.
 type CloudConfig struct {
-	SSHAuthorizedKeys []string `yaml:"ssh_authorized_keys"`
-	CoreOS            CoreOS   `yaml:"coreos"`
-	WriteFiles        []File   `yaml:"write_files"`
-	Hostname          string   `yaml:"hostname"`
-	Users             []User   `yaml:"users"`
-	ManageEtcHosts    EtcHosts `yaml:"manage_etc_hosts"`
-	NetworkConfigPath string   `yaml:"-"`
-	NetworkConfig     string   `yaml:"-"`
+	SSHAuthorizedKeys []string  `yaml:"ssh_authorized_keys"`
+	CoreOS            CoreOS    `yaml:"coreos"`
+	WriteFiles        []File    `yaml:"write_files"`
+	Hostname          string    `yaml:"hostname"`
+	Users             []User    `yaml:"users"`
+	ManageEtcHosts    EtcHosts  `yaml:"manage_etc_hosts"`
+	Internal          Internals `yaml:"-"`
```

Comment:
Seems like this could just be broken out entirely and passed (or its equivalent) as another arg to `Apply()` now